libs/core/src/scss/base/_functions.scss

```
@@ -231,9 +231,10 @@
     background-image: url('#{$svg}');
   } @else {
     // Icon from library
-    $icon-url: '/assets/kirby/icons/svg/#{$svg}.svg';
+    $icon-url: 'assets/kirby/icons/svg/#{$svg}.svg';
 
-    background-image: url('#{$icon-url}');
+    background-image: url('/#{$icon-url}'); // setting this as a fallback for older browsers that don't support "image-set()"
```

Consider simlifying:
```suggestion
    background-image: url('/#{$icon-url}'); // fallback for older browsers that don't support "image-set()"
```